src/common/components/cards/urls-card-row.tsx

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { NewTabLink } from 'common/components/new-tab-link';
+import * as React from 'react';
+
+import { CardRowProps } from 'common/configs/unified-result-property-configurations';
+import { NamedFC } from 'common/react/named-fc';
+import { SimpleCardRow } from './simple-card-row';
+
+export interface UrlsCardRowProps extends CardRowProps {
+    propertyData: string[];
```

Comment:
I know you can't change the name because it's set by the more general `CardRowProps` type, but I wonder if it might be helpful to clarify here exactly what this is supposed to contain (either with a comment or something like `propertyData: { urls: string[] }`

Reply:
I punted on creating a more descriptive propertyData Type in this PR in case the future work to integrate this lends itself better to nesting the array of URLs or not. Based on your comment, I'd prefer to just create the new type now vs adding a comment.

Reply:
Introduced `UrlsPropertyData` to improve clarity.